generator/README.md

```diff
@@ -82,14 +82,15 @@ modules:
                             # Required if context is configured on the device.
 
     lookups:  # Optional list of lookups to perform.
-              # This must only be used when the new index is unique.
+              # If keep_old is the default of false, this must only be used when the new index is unique.
 
       # If the index of a table is bsnDot11EssIndex, usually that'd be the label
       # on the resulting metrics from that table. Instead, use the index to
       # lookup the bsnDot11EssSsid table entry and create a bsnDot11EssSsid label
       # with that value.
       - old_indexes: [bsnDot11EssIndex]
         new_index: bsnDot11EssSsid
+        keep_old: false  # If true, don't delete old index label for this lookup.
```

Comment:
I think this should default to true. I tried running without the indices and it's a pain pit of duplicate label sets.

Reply:
When that happens the setting can be changed by the user, we shouldn't be adding labels blindly.

Reply:
We must add labels, rather than assume the new index is unique. The only true unique key for SNMP provides is the index. In order to provide end user safe defaults, this needs to be `true.`

Reply:
The default is not to have any lookups, which is perfectly safe.